home_style.css

```diff
@@ -0,0 +1,46 @@
+.title, h2{
+color:ghostwhite;
+}
+
+h1{
+    font-size: 90px;
+    text-align:end;
+}
+
+
+
+.container {
+    position: relative;
+    text-align: left;
+    color: white;
+    margin-top: 50px;
+}
+
+
+body{
+    background-color: #659dbd;
+    font-family: 'Gill Sans', 'Gill Sans MT', Calibri, 'Trebuchet MS', sans-serif;
+
+}
+
+ /*  Text to right */
+.rest{
+    float:right;
+    width: 48%;
+}
+
+ /* Button group to to of page */
+.btn-group button {
+    background-color: white; /* Green background */
```

Comment:
Should this be green or is it meant to be white? 

Comments are great for making things clear to us and anyone else who looks at our code. I would review any comment made on my own code and ask myself whether its absolutely necessary as its easy for our code sheets to look busy when we want them to look clean.